routes/web.php

```diff
@@ -22,3 +23,13 @@
 })->middleware(['auth'])->name('dashboard');
 
 require __DIR__.'/auth.php';
+
+Auth::routes();
+
+Route::get('/dashboard', [App\Http\Controllers\HomeController::class, 'index'])->name('dashboard');
+
+//routes for CRUD functionality
+Route::get('/show', [App\Http\Controllers\BalanceItemsController::class, 'show']);
+Route::post('/submit', [App\Http\Controllers\BalanceItemsController::class, 'store']);
+Route::post('/update', [App\Http\Controllers\BalanceItemsController::class, 'update']);
+Route::post('/delete', [App\Http\Controllers\BalanceItemsController::class, 'delete']);
```

Comment:
These should be restful.

Reply:
Yes, you're right - thank you.

I will update these routes so that they are proper and RESTful:

<!--StartFragment-->
Route::get('/show', [App\Http\Controllers\BalanceItemsController::class, 'show']);
Route::post('/submit', [App\Http\Controllers\BalanceItemsController::class, 'store']);
Route::put('/update', [App\Http\Controllers\BalanceItemsController::class, 'update']);
Route::delete('/delete', [App\Http\Controllers\BalanceItemsController::class, 'destroy']);
<!--EndFragment-->